velox/functions/sparksql/Arithmetic.h

```diff
@@ -262,6 +262,17 @@ struct Log1pFunction {
   }
 };
 
+template <typename T>
+struct LogarithmFunction {
+  FOLLY_ALWAYS_INLINE bool call(double& result, double a, double b) {
+    if (a <= 0 || b <= 0) {
+      return false;
+    }
+    result = std::log(b) / std::log(a);
```

Comment:
Can we add a test for `std::log(a) == 0`?

Reply:
Added, thanks